packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json

```diff
@@ -374,6 +374,17 @@
   "hosting_tab_DATABASES_configuration_check_quota_calculating": "Calcul en cours...",
   "hosting_tab_databases_get_error": "Une erreur est survenue lors de la récupération de vos bases de données.",
   "hosting_tab_databases_dumps_error_fetch": "Une erreur est survenue lors de la récupération de vos sauvegardes.",
+  "hosting_tab_DATABASES_configuration_copy_title_button": "Copier une base de données",
+  "hosting_tab_DATABASES_configuration_copy_title": "Copier une base de données",
+  "hosting_tab_DATABASES_configuration_copy_step1_question": "Où souhaitez-vous copier la base de donnée {{databaseName}} ?",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_database": "Base de données:",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_database_placeholder": "Sélectionner une base de données",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_webcloud_database": "Base de données Web Cloud:",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_webcloud_placeholder": "Sélectionner une base de données Web Cloud",
+  "hosting_tab_DATABASES_configuration_copy_step1_select_webcloud_database_placeholder": "Sélectionner une base de données",
+  "hosting_tab_DATABASES_configuration_copy_success": "Votre base de donnée à été copié avec succès",
```

Comment:
```suggestion
  "hosting_tab_DATABASES_configuration_copy_success": "Votre base de donnée à été copiée avec succès",
```

Reply:
"a été copiée" without "à" but with "a"

Reply:
Hello, you're right. It was written like this in the story and I just copied the text without checking. Will check next time.

Done